code/modules/clothing/masks/breath.dm

```diff
@@ -36,3 +36,12 @@
 	item_state = "m_mask"
 	permeability_coefficient = 0.01
 	equip_delay_other = 10
+
+/obj/item/clothing/mask/breath/vox

```

Comment:
I'd remove the mask. No point in it being a separate thing if it already molds itself to fit other species. Just use a regular one on Vox.

Reply:
iirc it has to be a unique thing because of the `gas_transfer_coefficient = 0.01`, which is not on the base breath mask.

Reply:
That variable literally does nothing.